feat(linux): DEV3701 - CTU uboot - add gpio command support

doron2020 opened the pull request:
gpio command can help in debugging gpio related features like leds

I enabled gpio cmd in defconfig
I also deleted calls to fumctions that are not supported in this boot branch like gpio_direction_input (it will not compile)

baruchsiach reviewed (changes requested):
Please mention in the commit log that Qualcomm GPIO driver provides partial implementation of the GPIO API that is missing `gpio_request()`, `gpio_direction_input()`, and `gpio_free()`. That is the reason we need to remove these calls from `cmd_gpio.c`.

Commit:
feat: DEV-3701: CTU uboot - add gpio command support

gpio command can help in debugging gpio related features like leds.
Qualcomm GPIO driver provides partial implementation of the GPIO API that is missing gpio_request(), gpio_direction_input(), and gpio_free(). That is the reason we need to remove these calls from cmd_gpio.c.

baruchsiach reviewed (approved):
Looks good.
